packages/js/e2e-utils/src/page-utils.js

```diff
@@ -1,7 +1,7 @@
 /**
  * External dependencies
  */
-import { pressKeyWithModifier } from '@wordpress/e2e-test-utils';
+ import { pressKeyWithModifier } from '@wordpress/e2e-test-utils';
```

Comment:
I think this space was unintentional?

Reply:
Indeed, I don't know how that space ended up there 😄 Thanks for spotting this!